test/Dapr.E2E.Test.App/Startup.cs

```diff
@@ -62,9 +63,17 @@ public void ConfigureServices(IServiceCollection services)
                 // Example of registering a "PlaceOrder" workflow function
                 options.RegisterWorkflow<string, string>("PlaceOrder", implementation: async (context, input) =>
                 {
+
+                    System.Threading.Thread.Sleep(10000); // sleep for 5s to allow the terminate command to come through
```

Comment:
This is sleeping for 10s. Is there no way we can hook into the terminate to know when it's happened? Or could we poll for status? I'm not a fan of sleeps in the tests if we can avoid them :)